init-mongo.sh

```
@@ -0,0 +1,31 @@
+mongod --replSet rs0 --bind_ip_all --dbpath /data/db &
```

Shellcheck (0.10.0)

[error] Tips depend on target shell and yours is unknown. Add a shebang or a 'shell' directive.

(SC2148)